misc/error_suppressions/tsan.txt

```diff
@@ -5,3 +5,4 @@ deadlock:tests/core/templates/test_command_queue.h
 deadlock:modules/text_server_adv/text_server_adv.cpp
 deadlock:modules/text_server_fb/text_server_fb.cpp
 race:modules/navigation/nav_map_3d.cpp
+race:thirdparty/thorvg/src/loaders/external_jpg/tvgJpgLoader.cpp
```

Comment:
I assume it is ok to suppress errors in third party code? Or does that still leave us vulnerable to crashes?

Reply:
The best I can do is try to file a bug report with @hermet at Thorvg